cve_bin_tool/cve_scanner.py

```diff
@@ -223,15 +228,26 @@ def get_cves(self, product_info: ProductInfo, triage_data: TriageData):
                     if duplicate_found:
                         continue
 
+                    # Check if we already found this CVE with a previous scan.
+                    # In that case we need to check where to get our triage info
+                    # from.
+                    # TODO: turn the list of CVEs into a set to avoid needing
+                    # the linear-time lookup.
+                    prev_cve = next(
+                        (
+                            cve
+                            for cve in prev_cves
+                            if cve.cve_number == row["cve_number"]
+                        ),
+                        None,
+                    )
+
                     triage = triage_data.get(row["cve_number"]) or triage_data.get(
                         "default"
                     )
-                    # Only scan cves if triage is not None.
-                    # Triage will only be None if triage_data don't have default attribute.
-                    # NOTE: Triage can be empty dictionary so checking `if triage:` won't suffice.
-                    if triage is not None:
+                    if prev_cve is None:
                         row_dict = dict(row)
-                        row_dict.update(triage)
```

Comment:
@terriko the plotly.js is coming from the: https://github.com/intel/cve-bin-tool/blob/2941eef9fcbf24cabea9838fb8282c68c3193290/triage.json#L76

triage.json being used in test_requirement.py as triage-input since the code mentioned tries to scan for the cves even if the triage data is present, apologies for not interpreting the changes in greater depth , cve_scanner is a little complex to read but this particular line is what i suspect is causing error hope it helps!